PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -996,7 +1122,8 @@ public void SetOption(PowerShellRawHostOptions options)
         }
 
         /// <summary>
-        /// Returns the connection info for the current runspace
+        /// Returns the connection info for the current runspace based on certain characteristics of the runspace. Note: the criteria for remote debug vs remote session

```

Comment:
There is almost no difference between the state of our runspace/debugger when we are in a remote session/remotely attached to a process. This is because to attach to a remote process, you must first enter a remote session. The only time a difference appears is when the debugger is stopped and the availability of the runspace changes to RemoteDebug. If debugger is not stopped though, then this difference goes away. Hence why in GetCallStack() I had to add the !(psobj.BaseObject is string) case to the remote session debugging version of parsing the call stack. The only other time that DebugScenario.RemoteAttach is used is during the debugger stop which is when we are actually able to use that difference between the two remote scenarios. Does that make sense?